cmd/src/users_clean.go

```diff
@@ -0,0 +1,202 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/sourcegraph/src-cli/internal/api"
+)
+
+func init() {
+	usage := `
+Examples:
+
+	$ src users clean -d 182
+
+`
+
+	flagSet := flag.NewFlagSet("clean", flag.ExitOnError)
+	usageFunc := func() {
+		fmt.Fprintf(flag.CommandLine.Output(), "Usage of 'src users %s':\n", flagSet.Name())
+		flagSet.PrintDefaults()
+		fmt.Println(usage)
+	}
+	var (
+		daysToDelete = flagSet.Int("d", 365, "Day threshold on which to remove users, defaults to 365")
```

Comment:
```suggestion
		daysToDelete = flagSet.Int("days", 365, "Days threshold on which to remove users, defaults to 365")
```

I think it is more important to be clear than short, unless we can provide both (long and short flag names).

Reply:
Changing this and also adding a minimum value of 60 per @ryphil's feedback